source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -33,34 +33,34 @@ Locate the following control system components and tools
       - 2x PWM y-cables
       - 120A Circuit breaker
       - 4x 40A Circuit breaker
-      - 6 AWG Red wire
-      - 10 AWG Red/Black wire
-      - 18 AWG Red/Black wire
-      - 22AWG yellow/green twisted CAN cable
-      - 16x 10-12 AWG  (yellow) ring terminals
+      - 6 AWG (16 mm^2) Red wire
```

Comment:
We can use the ``.. math::`` directive to properly wrap squared.

Reply:
fixed